README.md

```diff
@@ -22,49 +34,149 @@ You can do it manually following the instructions from [rust-lang.org](https://w
+   `./<build_repository>/core/miner/main`
+8. next steps are similar to common miner and node establishing which can be found at **Fuhon node establishing** and **Fuhon miner establishing**
+
+9. To finish your building you may also want to add your github token, In such case you need to set up two environment variables:
 ```
 GITHUB_HUNTER_USERNAME=<github account name>
 GITHUB_HUNTER_TOKEN=<github token>
 ```
-To generate github token follow the [instructions](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line). Make sure `read:packages` and `write:packages` permissions are granted (step 7 in instructions).
+To generate github token follow the [instructions](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line). Make sure `read:packages` and `write:packages` permissions are granted.
 
-This project is can be built with
 
-```
-mkdir build && cd build
-cmake -DCLANG_TIDY=ON ..
-make -j
-```
 
-It is suggested to build project with clang-tidy checks, however if you wish to omit clang-tidy step, you can use `cmake ..` instead.
+## Configuration Lotus-Fuhon
+### Lotus-node and Fuhon-miner
+1. To start up a lotus node at first you need to get [lotus executable](https://github.com/filecoin-project/lotus/releases/ "lotus releases")
+2. You can run your lotus node with the next command: `$lotus --repo <node working directory> daemon --genesis genesis.car --import-key PRESEAL/pre-seal-MINER.key --profile bootstrapper`, more about running lotus software can be found in [filecoin-docs](https://docs.filecoin.io/get-started/lotus/installation "lotus filecoin documentation").
 
-Tests can be run with: 
-```
-cd build
-ctest
-```
+### Fuhon-node and Lotus-miner
+1. Establishing of a Fuhon node is described in **Fuhon miner establishing**
+2. Lotus miner can be established by the following way:
+    - to begin with you need you own bls wallet(described in **Working with Lotus-CLI**)
+    - run your miner with command: `lotus-miner --repo <your node running directory> --miner-repo <minner future working directory> init --nosync --owner <your bls wallet public address> --sector-size <Size of sectors that will be sealed>`
+    - to know more about lotus software establishing, read [filecoin-docs](https://docs.filecoin.io/get-started/lotus/installation/#minimal-requirements "lotus establishing guide")
 
-### CodeStyle
+## Docker configuration
+    Will be soon!
 
-We follow [CppCoreGuidelines](https://github.com/isocpp/CppCoreGuidelines).
+## Your new Filecoin network start
 
-Please use clang-format 11.0.0 with provided [.clang-format](.clang-format) file to autoformat the code.  
+To run a new filecoin network, you should run at least two nodes, that will have comunication with each other and a miner (miners are representation of the network power, so more miners connected and sealed, more power your network will have).
```

Comment:
network may have single node, but requires at least one genesis miner (to generate blocks)